app/js/services/auth.js

```diff
@@ -0,0 +1,82 @@
+import Cookies from 'js-cookie'
+import axios from 'axios'
+import { Promise } from 'es6-promise'
+import { CP_ADMIN_HOST, CP_ANALYTICS_HOST } from '../constants/app_constants'
+import { routeUtils } from '../helpers/router'
+
+const LOGIN_URL = CP_ADMIN_HOST + '#/auth/sign_in?return_url=' + CP_ANALYTICS_HOST
+const SESSION_PING_URL = CP_ADMIN_HOST + 'api/v1/session'
+const REFRESH_TOKEN_URL = CP_ADMIN_HOST + 'api/v1/api_session/refresh'
+
+const Auth = {
+  init () {
+    this.addXHRInterceptor()
+    this.initSession()
+  },
+
+  addXHRInterceptor () {
+    axios.interceptors.request.use(
+      this.addXHRHeaders.bind(this),
+      (error) => { return Promise.reject(error) }
```

Comment:
The `=>` has an implicit return if it's a single line. So this line is exactly the same as 
```
(error) => Promise.reject(error)
```

You just need the curly braces and `return` keyword if it's more than one line.